feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeViewModel.kt

```diff
@@ -76,23 +65,24 @@ class OfficeViewModel() : ViewModel() {
  *
  * @author Andres Ivan Medina
  */
-val officeInfoResponse: DataState<OfficeModel> = DataState.Success(
-    OfficeModel(
+val officeResponse: DataState<Office> = DataState.Success(
+    Office(
         "4950 Gaidrew, Alpharetta, GA, 30022 ",
```

Comment:
add named param address = ....... 4950 Gaidrew, Alpharetta, GA, 30022. Note: delete extra space at end of the string.
same for schedule = 9AM-6PM